.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature request
+about: Request a new feature to the plugin
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+## Feature description
+<!--- Describe your idea. --->
```

Comment:
Do we want to use comments? The final issue should look cleaner. I have no strong feelings. We could use them in the bug report template too. wdyt?

Reply:
+1

Reply:
Pls update bug template indeed

Reply:
Oh, there's even #1380 for that